terraform-google-{{cookiecutter.module_name}}/test/integration/simple_example/simple_example_test.go

```diff
@@ -0,0 +1,40 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package multiple_buckets
+
+import (
+	"testing"
+
+	"github.com/GoogleCloudPlatform/cloud-foundation-toolkit/infra/blueprint-test/pkg/gcloud"
+	"github.com/GoogleCloudPlatform/cloud-foundation-toolkit/infra/blueprint-test/pkg/tft"
+	"github.com/GoogleCloudPlatform/cloud-foundation-toolkit/infra/blueprint-test/pkg/utils"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestSimpleExample(t *testing.T) {
+	example := tft.NewTFBlueprintTest(t)
+
+	example.DefineVerify(func(assert *assert.Assertions) {
+		example.DefaultVerify(assert)
+
+		projectID := example.GetStringOutput("project_id")
+		gcloudArgs := gcloud.WithCommonArgs([]string{"--project", projectID, "--format", "json"})
+		services := gcloud.Run(t, "services list", gcloudArgs).Array()
```

Comment:
As a note, I think I have found it to be confusing to store `gcloudArgs` as a variable instead of just using it inline the one time we reference it.

Reply:
Sounds good, I tend to use gcloudArgs as a var if there is more than 1 gcloud.Run.